From: Christian Marillat <marillat@deb-multimedia.org>
Date: Thu, 6 Jul 2017 15:59:17 +0200
Subject: Plugin path OFX

Need to patch the Natron plug-in path into the embedded OpenFX API.
According to Filesystem-Hierarchy-Standard, these go into /user/lib/...
---

Index: b/OpenFX/HostSupport/src/ofxhPluginCache.cpp
===================================================================
--- a/OpenFX/HostSupport/src/ofxhPluginCache.cpp
+++ b/OpenFX/HostSupport/src/ofxhPluginCache.cpp
@@ -314,7 +314,7 @@ PluginCache::PluginCache()
   _pluginPath.push_back("C:\\Program Files\\Common Files\\OFX\\Plugins");
 #endif
 #if defined(__linux__) || defined(__FreeBSD__)
-  _pluginPath.push_back("/usr/OFX/Plugins");
+  _pluginPath.push_back("/usr/lib/OFX/Plugins");
 #endif
 #if defined(__APPLE__)
   _pluginPath.push_back("/Library/OFX/Plugins");
Index: b/OpenFX/Support/Plugins/Makefile.master
===================================================================
--- a/OpenFX/Support/Plugins/Makefile.master
+++ b/OpenFX/Support/Plugins/Makefile.master
@@ -55,7 +55,7 @@ ifeq ($(OS:MINGW%=MINGW),MINGW)
   PLUGINPATH=C:\\Program Files\\Common Files\\OFX\\Plugins
 endif
 ifeq ($(OS),Linux)
-  PLUGINPATH=/usr/OFX/Plugins
+  PLUGINPATH=/usr/lib/OFX/Plugins
 endif
 ifeq ($(OS),FreeBSD)
   PLUGINPATH=/usr/OFX/Plugins
Index: b/OpenFX-IO/openfx/HostSupport/src/ofxhPluginCache.cpp
===================================================================
--- a/OpenFX-IO/openfx/HostSupport/src/ofxhPluginCache.cpp
+++ b/OpenFX-IO/openfx/HostSupport/src/ofxhPluginCache.cpp
@@ -314,7 +314,7 @@ PluginCache::PluginCache()
   _pluginPath.push_back("C:\\Program Files\\Common Files\\OFX\\Plugins");
 #endif
 #if defined(__linux__) || defined(__FreeBSD__)
-  _pluginPath.push_back("/usr/OFX/Plugins");
+  _pluginPath.push_back("/usr/lib/OFX/Plugins");
 #endif
 #if defined(__APPLE__)
   _pluginPath.push_back("/Library/OFX/Plugins");
Index: b/OpenFX-IO/openfx/Support/Plugins/Makefile.master
===================================================================
--- a/OpenFX-IO/openfx/Support/Plugins/Makefile.master
+++ b/OpenFX-IO/openfx/Support/Plugins/Makefile.master
@@ -55,7 +55,7 @@ ifeq ($(OS:MINGW%=MINGW),MINGW)
   PLUGINPATH=C:\\Program Files\\Common Files\\OFX\\Plugins
 endif
 ifeq ($(OS),Linux)
-  PLUGINPATH=/usr/OFX/Plugins
+  PLUGINPATH=/usr/lib/OFX/Plugins
 endif
 ifeq ($(OS),FreeBSD)
   PLUGINPATH=/usr/OFX/Plugins
